lib/std/room.c

```diff
@@ -77,6 +78,17 @@ int query_weather(void) {
    return weather;
 }
 
+void set_pk(int flag) {
+   pk = flag;
+}
+
+int query_pk(void) {
+  if (!pk) {
```

Comment:
Maybe just a style preference but the `if()` is not necessary since `pk` is an integer.

Reply:
I'm leaving it in because I'm debating adding in additional conditions such as a level difference between killers even if pk is set, alternatively I'm considering a flag on players that makes them pk safe, or adding in conditions where admins can't be killed etc.

By default PK is enabled, and this was my quick fix on it.

I am open to removing the brackets though if you feel its not needed

Reply:
No change requested. Thanks for sharing your thoughts.